kyo-prelude/jvm/src/main/scala/kyo2/kernel/Platform.scala

```diff
@@ -0,0 +1,25 @@
+package kyo2.kernel
+
+import kyo2.kernel.Mode
+import scala.concurrent.ExecutionContext
+
+object Platform:
+
+    val mode: Mode =
+        Option(System.getProperty("kyo.kernel.Platform.mode"))
+            .map(Mode.valueOf).getOrElse {
+                if isDebugEnabled then
+                    Mode.Debugging
+                else
+                    Mode.Development
```

Comment:
The idea was to have `Development` as the default and `Debugging` as a specialization of it when the development environment has the debug port enabled. But reconsidering it, it seems a bit dangerous that `Debugging` might have behaviors that make only sense in a development environment but can be enabled in production just by setting the debug port configurations. Perhaps we could remove `Mode.Debugging` since it's still possible to check if the debug port is enabled via `Platform.isDebugEnabled`.